packages/slate-react/src/components/editor.js

```diff
@@ -145,11 +145,16 @@ class Editor extends React.Component {
   }
 
   /**
-   * When the component first mounts, flush any temporary changes.
+   * When the component first mounts, flush any temporary changes,
+   * and then, focus the editor if `autoFocus` is set.
    */
 
   componentDidMount = () => {
     this.flushChange()
```

Comment:
I fixed the bug by hoisting `.focus` to Editor's componentDidMount.

But I don't understand why we need to cache initial value and then flush it again. @ianstormtaylor